src/main/steps/urls.ts

```diff
@@ -243,6 +243,10 @@ export const C100_C1A_SAFETY_CONCERNS_OTHER_CONCERNS_DRUGS: PageLink = `${C100_C
 /** @C100 Document Submission */
 export const C100_DOCUMENT_SUBMISSION: PageLink = `${C100_URL}/document-submission`;
 
+/** @C100 Alternative Rsesolution */
+export const C100_ALTERNATIVE_RESOLUTION: PageLink = `${C100_URL}/alternative-resolution`;
```

Comment:
@kumarmadurai This is part of screening questions, we need to add these changes inside /screening-questions folder & in URL.

Reply:
@yogendraU Can you please add your inputs

Reply:
yes. it should be part of /screening-questions